data-service/src/contract-tests/java/uk/gov/gchq/palisade/contract/data/rest/RestContractTest.java

```diff
@@ -0,0 +1,135 @@
+import uk.gov.gchq.palisade.contract.data.config.DataTestConfiguration;
+import uk.gov.gchq.palisade.contract.data.config.mock.AuditServiceMock;
+import uk.gov.gchq.palisade.contract.data.config.mock.PalisadeServiceMock;
+import uk.gov.gchq.palisade.contract.data.config.web.DataClient;
+import uk.gov.gchq.palisade.contract.data.config.web.DataClientWrapper;
+import uk.gov.gchq.palisade.contract.data.model.Employee;
+import uk.gov.gchq.palisade.data.serialise.AvroSerialiser;
+import uk.gov.gchq.palisade.jsonserialisation.JSONSerialiser;
+import uk.gov.gchq.palisade.reader.common.DataFlavour;
+import uk.gov.gchq.palisade.resource.impl.FileResource;
+import uk.gov.gchq.palisade.service.data.DataApplication;
+import uk.gov.gchq.palisade.service.data.request.ReadRequest;
+import uk.gov.gchq.palisade.service.data.service.DataService;
+import uk.gov.gchq.palisade.util.ResourceBuilder;
+
+import java.io.IOException;
+import java.nio.file.Paths;
+import java.util.Collections;
+import java.util.Map;
+import java.util.Set;
+import java.util.stream.Collectors;
+
+import static org.assertj.core.api.Assertions.assertThat;
+import static org.junit.Assert.assertNotEquals;
+import static org.junit.Assert.assertNotNull;
+
+@EnableFeignClients(basePackageClasses = {DataClient.class})
+@Import(DataTestConfiguration.class)
+@SpringBootTest(classes = DataApplication.class, webEnvironment = WebEnvironment.DEFINED_PORT)
+class RestContractTest {
+    private static final ObjectMapper MAPPER = JSONSerialiser.createDefaultMapper();
+
+    private static final String CURRENT_PATH = Paths.get("src/contract-tests/resources/data/employee_file0.avro").toUri().toString();
+    private static final FileResource RESOURCE = ((FileResource) ResourceBuilder.create(CURRENT_PATH))
+            .type(Employee.class.getTypeName())
+            .serialisedFormat("avro");
+
+    @Autowired
+    private Map<String, DataService> serviceMap;
+    @Autowired
+    private DataClientWrapper client;
+
+    private final WireMockRule auditMock = AuditServiceMock.getRule();
+    private final WireMockRule palisadeMock = PalisadeServiceMock.getRule();
+
+    private AvroSerialiser<Employee> avroSerialiser;
+
+    @BeforeEach
+    void setUp() throws JsonProcessingException {
+        AuditServiceMock.stubRule(auditMock, MAPPER);
+        auditMock.start();
+
+        PalisadeServiceMock.stubRule(palisadeMock, MAPPER, RESOURCE);
+        palisadeMock.start();
+
+        MAPPER.configure(DeserializationFeature.FAIL_ON_UNKNOWN_PROPERTIES, false);
+        avroSerialiser = new AvroSerialiser<>(Employee.class);
+    }
+
+    @AfterEach
+    void tearDown() {
+        auditMock.stop();
+        auditMock.resetAll();
+
+        palisadeMock.stop();
+        palisadeMock.resetAll();
+    }
+
+    @Test
+    void contextLoads() {
+        assertNotNull(serviceMap);
+        assertNotEquals(serviceMap, Collections.emptyMap());
+    }
+
+    @Test
+    void testIsUp() {
```

Comment:
Test is only testing for HTTP 200, but I think that the test is meant to evaluate the status is up, i.e. {status:UP}

Reply:
all we need to test is the status as this is what k8s uses, not status:UP